app/main.py

```diff
@@ -1,25 +1,69 @@
+from __future__ import annotations
 from abc import ABC
 
 
 class IntegerRange:
-    pass
+    def __init__(self,
+                 min_amount: int,
+                 max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __get__(self, instance: Visitor, owner: type) -> int:
+        return getattr(instance, self.proteted_name)
+
+    def __set__(self,
+                instance: Visitor,
+                value: int) -> None:
+        if not self.min_amount <= value <= self.max_amount:
+            raise ValueError("Visitor can't use the slide")
```

Comment:
You should use more descriptive message here. It is an descriptor, so you want to add a message that will describe that ValueError was raised because of your condition "if not self.min_amount <= value <= self.max_amount"